sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -1159,3 +1191,122 @@ def _check_consistency(self, y_test, y_pred):
 
         # check consistency of forecast hierarchy with training data
         assert set(y_test.index.names).issubset(y_pred.index.names)
+
+
+def _get_expected_columns(y_test):
+    if isinstance(y_test, pd.Series):
+        if hasattr(y_test, "name") and y_test.name is not None:
+            return [y_test.name]
+        else:
+            return [0]
+    else:
+        return y_test.columns
+
+
+def _check_predict_intervals(pred_ints, y_test, fh, coverage):
+    """Check expected interval prediction output."""
+    # check expected type
+    valid, msg, _ = check_is_mtype(
+        pred_ints,
+        mtype="pred_interval",
+        scitype="Proba",
+        return_metadata=True,
+        msg_return_dict="list",
+    )  # type: ignore
+    assert valid, msg
+
+    # check index (also checks forecasting horizon is more than one element)
+    cutoff = get_cutoff(y_test, return_index=True)
+    index_pred = pred_ints.iloc[
+        : 1 if isinstance(fh, int) else len(fh)
+    ].index.get_level_values(len(pred_ints.index.names) - 1)
```

Comment:
would it be the same as `get_level_values(-1)`?

Reply:
changed to `get_level_values(-1)`